### DISCUSSION TITLE
Google Search button does not show on published pages, but does on preview pages

### DISCUSSION THREAD

**Discussion by zipswich** [GetPublii/Publii#1459] [Help] [answered]
I followed [the official instructions to enable Google Search](https://getpublii.com/docs/google-custom-search.html).  
The preview has the search buttons:
![image](https://github.com/GetPublii/Publii/assets/6095671/bb7b3ccf-efe8-47e4-8add-07faaa9db756)

However, the published pages do not have any search buttons. 
Could anyone offer a tip on how to diagnose this?

**Comment by bobmitro** [comment]
any URL?

**Reply by zipswich** [reply]
@bobmitro Do you mean the URL of the published pages?

**Reply by bobmitro** [reply]
Yeap

**Reply by zipswich** [reply]
I have just emailed it to you.

**Reply by zipswich** [reply]
I see svg-map.svg in
  home/assets/svg
not in 
  home/blog/assets/svg
![image](https://github.com/GetPublii/Publii/assets/6095671/eba822d6-7b66-41cc-b6f8-9fdd5d41a357)

I do not see the following directory:
  home/blog/

Here is the structure under home:
![image](https://github.com/GetPublii/Publii/assets/6095671/8a791a57-3d8e-447b-b121-e00f0599cd58)

**Comment by bobmitro** [comment]
Update **Remote Path** in your server settings. It should help.

**Reply by zipswich** [reply]
I have just done the following according to your suggestion:
1. update the Remtoe path to "\blog" from "\virtual_home".
2. went into IIS and changed the biding of the virtual directory blog to "..\blog" from "...\virtual_home" and deleted "...\virtual_home"

The pages look good like before, however they still miss the search buttons. 
![image](https://github.com/GetPublii/Publii/assets/6095671/f4352b29-25f0-4e5a-947e-290ecbed1e36)

**Comment by bobmitro** [comment, marked as answer]
The "svg-map.svg" file is still missing on the production site. However, the scripts located in the same directory (/assets) are there. So, there might be some reason why the SVG file isn't uploading or is blocked on the server. Try manually copying the file to the server.

**Reply by zipswich** [reply]
All the screenshots in my previous posts are from the server. 
Let me do some experiments to see if I can find a clue.

**Reply by zipswich** [reply]
Just did an experiment:
I put file autosave.jpg in the directory /assets/svg/
.../blog/assets/svg/autosave.jpg

When I browse https://home/blog/assets/svg/autosave.jpg and the image shows.

I assume this means svg-map.svg is somehow blocked.

**Reply by zipswich** [reply]
[This ](https://stackoverflow.com/a/12328668/355456)did it for svg.

**Reply by zipswich** [reply]
All good now. Thanks a lot for the help.
